.eslintrc.js

```diff
@@ -1,23 +1,66 @@
-{
+// Rules which have been enforced in configuration upgrades and flag issues in existing code.
+// We need to consider whether to disable those rules permanently, or fix the issues.
```

Comment:
For everything in this list, we’ll need follow-up work to either disable the rule in our shared configuration, or refactor the code and enforce it here.